wcfsetup/install/files/lib/system/form/builder/field/UploadFormField.class.php

```diff
@@ -345,13 +345,22 @@ public function getFieldHtml()
     public function updatedObject(array $data, IStorableObject $object, $loadValues = true)
     {
         if ($loadValues) {
-            // first check, whether an getter for the field exists
-            if (\method_exists($object, 'get' . \ucfirst($this->getObjectProperty()) . 'UploadFileLocations')) {
+            $expectUploadFile = false;
+
+            // first check, whether a getter for the field exists; preferring get{$fieldName}UploadFiles() before get{$fieldName}UploadFileLocations()
+            if (\method_exists($object, 'get' . \ucfirst($this->getObjectProperty()) . 'UploadFiles')) {
+                $value = \call_user_func([
+                    $object,
+                    'get' . \ucfirst($this->getObjectProperty()) . 'UploadFiles',
+                ]);
+                $method = "method '" . \get_class($object) . "::get" . \ucfirst($this->getObjectProperty()) . "UploadFiles()'";
+                $expectUploadFile = true;
+            } elseif (\method_exists($object, 'get' . \ucfirst($this->getObjectProperty()) . 'UploadFileLocations')) {
                 $value = \call_user_func([
                     $object,
                     'get' . \ucfirst($this->getObjectProperty()) . 'UploadFileLocations',
                 ]);
-                $method = "method '" . \get_class($object) . "::get" . \ucfirst($this->getObjectProperty()) . "UploadFileLocations()'";
+                $method = "method '" . \get_class($object) . '::get' . \ucfirst($this->getObjectProperty()) . "UploadFileLocations()'";
```

Comment:
Why this change here?

Reply:
I assume I copied or moved the part and the IDE did some "improvements" since it's configured to prefer `'`.